src/Sulu/Bundle/ContactBundle/Resources/config/lists/accounts.xml

```diff
@@ -247,12 +254,25 @@
             <field-name>id</field-name>
             <entity-name>SuluTagBundle:Tag</entity-name>
 
-            <joins>
-                <join>
-                    <entity-name>SuluTagBundle:Tag</entity-name>
-                    <field-name>%sulu.model.account.class%.tags</field-name>
-                </join>
-            </joins>
+            <joins ref="tags"/>
         </group-concat-property>
+
+        <property
+            name="tagId"
+            visibility="never"
+            translation="sulu_tag.tags"
+        >
+            <field-name>id</field-name>
+            <entity-name>SuluTagBundle:Tag</entity-name>
+
+            <joins ref="tags"/>
+
+            <filter type="selection">
```

Comment:
we cannot do this for `categories` at the moment because the `SelectionFieldFilterType` does not send any locale to the list api

Reply:
TODO: adjust the `SelectionFieldFilterType` to be consistent with the `Selection`:
https://github.com/sulu/sulu/blob/33b1356d979ac5095ff9f4d32be0ad0aac24856e/src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/Selection.js#L220-L224

Reply:
This change depends on https://github.com/sulu/sulu/pull/5805, because the `locale` parameter of the `SearchStore` is added in https://github.com/sulu/sulu/pull/5805